frontend/src/models/types.ts

```diff
@@ -93,6 +93,7 @@ export interface User {
   sessionId: SessionId;
   onAssessment: boolean;
   currentInterventionOrAssessment: string;
+  daysActive: Array<string>;
```

Comment:
Can we use the literal syntax here? 
```suggestion
  daysActive: string[];
```